src/components/product-details/product-details.hbs

```diff
@@ -59,13 +59,11 @@
             <label for="color-red" class="radio__label"> Red</label>
           </div>
           <div class="product-description__radio-group">
-            <input type="radio" class="radio__input" value="orange" name="color" id="color-orange"
-              aria-describedby="color">
+            <input type="radio" class="radio__input" value="orange" name="color" id="color-orange" aria-describedby="color">
```

Comment:
@aevanson what is the aria-describedby pointing to? it should be an id of content that is different from, and in addition to, the label.

Reply:
Hmm, good point. Might be something that was lost after hackathon refactoring. I'll add a new defect to clean up the aria attributes for the product form on the PDP.